astropy/utils/iers/tests/test_iers_b_into_auto.py

```diff
@@ -0,0 +1,128 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import os
+import urllib.request
+import warnings
+
+import pytest
+import numpy as np
+from numpy.testing import assert_equal
+
+from astropy.tests.helper import assert_quantity_allclose
+from astropy.utils.data import download_file, clear_download_cache
+from astropy.utils.iers import (
+    IERS_A, IERS_Auto, IERS_B, IERS_B_URL, IERS_B_FILE
+)
+from astropy import units as u
+from astropy.table import QTable
+from astropy.time import Time, TimeDelta
+
+try:
```

Comment:
Huh. I thought I'd flake8'd the file. It's there because I was poking around trying to figure out which values agreed with which and why, but I might have deleted the experiments as not useful tests. I'll check.